scripts/deploy.sh

```
@@ -0,0 +1,79 @@
+#!/bin/bash
+set -euo pipefail
+
+# Variables
+AWS_REGION="${AWS_REGION:-ap-northeast-2}"
+ECR_REGISTRY="${ECR_REGISTRY}"
```

Shellcheck (0.11.0)

[info] This variable is assigned to itself, so the assignment does nothing.

(SC2269)